.github/workflows/gradle.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        idea: [ '2024.2', '2024.2.2', '2024.3' ]
+        idea: [ '2024.3', '2025.1' ]
```

Comment:
is it worth setting `251-EAP-SNAPSHOT` while waiting for the release. That way it can be used when testing other branches